spec/integration/_dev/benchmarks/spec.yml

```diff
@@ -0,0 +1,33 @@
+spec:
+  additionalContents: false
+  developmentFolder: true
+  contents:
+    - description: Folder containing end-to-end benchmarks scenarios.
+      type: folder
+      name: e2e
+      required: false
+      additionalContents: true
+      contents:
+        - description: Folder containing deployable services for benchmarks.
+          type: folder
+          name: deploy
+          additionalContents: true
+          required: false
+          contents:
+            - description: Folder containing docker compose files for benchmarks.
+              type: folder
+              name: docker
+              additionalContents: true
+              required: false
+              contents:
+                - description: Docker Compose configuration file for deploying benchmarks additional services.
+                  type: file
+                  name: "docker-compose.yml"
+                  contentMediaType: "application/x-yaml"
+                  required: true
+        - description: A benchmark scenario.
+          type: file
+          pattern: '^.+\.yml$'
```

Comment:
As in the test package, I see that the files is named `tcp-benchmark.yml`.
Would it be required some kind of prefixes or suffixes in these files ? Or could they have any name?

Reply:
I was not thinking about limiting that to any particular naming conventions, do you think could be benefficial?

Reply:
I just wanted to check whether or not they should follow a pattern. If it is not needed, I think we could keep it as it is :+1: